util/converter/converter.go

```diff
@@ -55,6 +71,20 @@ var bufioPool = sync.Pool{
 	},
 }
 
+func rewriteEpochInTarHeader(epoch time.Time) tarconverter.HeaderConverter {
+	return func(hdr *tar.Header) {
+		if hdr.ModTime.After(epoch) {
```

Comment:
Rules for rewriting epoch should only apply if old timestamp is after epoch value.

Reply:
The current code conforms to that AFAICS